vllm/v1/kv_cache_interface.py

```diff
@@ -159,6 +159,7 @@ def max_memory_usage_bytes(self, vllm_config: VllmConfig) -> int:
 class MambaSpec(KVCacheSpec):
     shapes: tuple[tuple[int, ...], ...]
     dtype: torch.dtype
+    multiple_of: Optional[int]
```

Comment:
Why is it a `multiple_of`? To my understanding, we have to set `page_size_bytes` of mamba layer exactly to that of attention layers instead of `multiple_of` that size.